dll/win32/mshtml/olecmd.c

```diff
@@ -241,6 +241,13 @@ static HRESULT exec_print(HTMLDocument *This, DWORD nCmdexecopt, VARIANT *pvaIn,
         return S_OK;
     }
 
+#ifdef __REACTOS__
+    // returning here fixes CORE-16884. Maybe use this until printing works.
+    ERR("Sorry, ReactOS does not support printing yet.\n");
```

Comment:
```suggestion
    ERR("HACK; Aborting print, to work around CORE-16884\n");
```

Reply:
I do give a thumbs up to Serges proposal, because I think mentioning the CORE-ID in the dbg print makes sense, and I do like that he omitted the final '.' character in the dbg print. .
But my thumbs up is only without using the word "hack". So my proposal would be:
 ERR("Aborting print, to work around CORE-16884\n");

Reply:
Ill change it to this! thanks